src/nwp_consumer/internal/entities/parameters.py

```diff
@@ -0,0 +1,357 @@
+                                "Defined as the horizontal speed of "
+                                "the wind in the northward direction.",
+                    units="m/s",
+                    limits=ParameterLimits(upper=150, lower=-150),
+                    alternate_shortnames=["v200"],
+                )
+            case self.SNOW_DEPTH_GL.name:
+                return ParameterData(
+                    name=str(self),
+                    description="Depth of snow on the ground.",
+                    units="m",
+                    limits=ParameterLimits(upper=12, lower=0),
+                    alternate_shortnames=["sd", "sdwe"],
+                )
+            case self.CLOUD_COVER_HIGH.name:
+                return ParameterData(
+                    name=str(self),
+                    description="Fraction of grid square covered by high-level cloud. "
+                                "Defined as the ratio of "
+                                "the area of the grid square covered by high-level (>6km) cloud "
+                                "to the square's total area.",
+                    units="UI",
+                    limits=ParameterLimits(upper=1, lower=0),
+                    alternate_shortnames=["hcc"],
+                )
+            case self.CLOUD_COVER_MEDIUM.name:
+                return ParameterData(
+                    name=str(self),
+                    description="Fraction of grid square covered by medium-level cloud. "
+                                "Defined as the ratio of "
+                                "the area of the grid square covered by medium-level (2-6km) cloud "
+                                "to the square's total area.",
+                    units="UI",
+                    limits=ParameterLimits(upper=1, lower=0),
+                    alternate_shortnames=["mcc"],
+                )
+            case self.CLOUD_COVER_LOW.name:
+                return ParameterData(
+                    name=str(self),
+                    description="Fraction of grid square covered by low-level cloud. "
+                                "Defined as the ratio of "
+                                "the area of the grid square covered by low-level (<2km) cloud "
+                                "to the square's total area.",
+                    units="UI",
+                    limits=ParameterLimits(upper=1, lower=0),
+                    alternate_shortnames=["lcc"],
+                )
+            case self.CLOUD_COVER_TOTAL.name:
+                return ParameterData(
+                    name=str(self),
+                    description="Fraction of grid square covered by any cloud. "
+                                "Defined as the ratio of "
+                                "the area of the grid square covered by any cloud "
+                                "to the square's total area.",
+                    units="UI",
+                    limits=ParameterLimits(upper=1, lower=0),
```

Comment:
are any in percentage? i thnk ive seen cloud cover go from 0-100 before, but perhaps you've convered this